Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -0,0 +1,207 @@
+                    completion: { finished in
+                        self.interstitialSpinnerView.removeFromSuperview()
+                        self.interstitialSpinnerView.stopAnimating()
+                })
+            }
+        }
+    }
+
+    // The view shown while the content is loading in the background web view.
+    private var interstitialView: UIView!
+    private var interstitialSpinnerView: UIActivityIndicatorView!
+    private var interstitialErrorView: UILabel!
+
+    // The web view that displays content.
+    var webView: WKWebView!
+
+    private func startLoading(timeout: Double = DefaultTimeoutTimeInterval) {
+        if self.isLoaded {
+            return
+        }
+        if timeout > 0 {
+            self.timer = NSTimer.scheduledTimerWithTimeInterval(timeout, target: self, selector: "SELdidTimeOut", userInfo: nil, repeats: false)
+        } else {
+            self.timer = nil
+        }
+        self.webView.loadRequest(NSURLRequest(URL: url))
+        self.interstitialSpinnerView.startAnimating()
+    }
+
+    init(backgroundColor: UIColor = UIColor.whiteColor(), title: NSAttributedString? = nil) {
+        interstitialBackgroundColor = backgroundColor
+        settingsTitle = title
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // This background agrees with the web page background.
+        // Keeping the background constant prevents a pop of mismatched color.
+        view.backgroundColor = interstitialBackgroundColor
+
+        self.webView = makeWebView()
+        view.addSubview(webView)
+
+        // Destructuring let causes problems.
+        let ret = makeInterstitialViews()
+        self.interstitialView = ret.0
+        self.interstitialSpinnerView = ret.1
+        self.interstitialErrorView = ret.2
+
+        view.addSubview(interstitialView)
+        layoutInnerViews()
+        startLoading()
+    }
+
+    func makeWebView() -> WKWebView {
+        let config = WKWebViewConfiguration()
+        let webView = WKWebView(
+            frame: CGRect(x: 0, y: 0, width: 1, height: 1),
+            configuration: config
+        )
+        webView.navigationDelegate = self
+        return webView
+    }
+
+    private func makeInterstitialViews() -> (UIView, UIActivityIndicatorView, UILabel) {
+        let view = UIView()
+
+        // Keeping the background constant prevents a pop of mismatched color.
+        view.backgroundColor = interstitialBackgroundColor
+
+        let spinner = UIActivityIndicatorView(activityIndicatorStyle: UIActivityIndicatorViewStyle.Gray)
+        view.addSubview(spinner)
+
+        let error = UILabel()
+        if let settingsTitle = settingsTitle {
+            error.text = String(format: NSLocalizedString("Could not connect to %s",
```

Comment:
Can you replace this string with the one from the TODO placeholder? We won't be able to localize settingsTitle and inject it into the string.